pages/views/sitemap.njk

```diff
@@ -0,0 +1,17 @@
+---
+permalink: "/sitemap.xml"
+eleventyExcludeFromCollections: true
+---
+
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+{% for page in collections.all %}
+{% if page.data.ignore != true %}
+  <url>
+    <loc>{{ site.url }}{{ page.url.replace(r/\/$/, ".html") if page.data.tags else page.url }}</loc>
+    <lastmod>{{ page.date.toISOString() }}</lastmod>
+    <priority>{{ "1.00" if page.url == "/" else "0.90" }}</priority>
```

Comment:
It is enough to use values up to one decimal place.
Moreover, it is enough to specify the priority for only one root page. From the resulting file, it will be clear to the search engine crawler that you give priority only to the root page. The rest of the pages will be considered equivalent, so you can use the default value for them and therefore not set a value at all. In fact, this priority will not affect the search results in any way. Only for the order in which the pages are scanned.